examples/authentication-rs/ta/build.rs

```diff
@@ -58,5 +82,12 @@ fn main() -> std::io::Result<()> {
 
     println!("cargo:rustc-link-search={}", search_path.display());
     println!("cargo:rustc-link-lib=static=utee");
+    println!("cargo:rustc-link-lib=static=utils");
+    println!("cargo:rustc-link-arg=-Tta.lds");
+    println!("cargo:rustc-link-arg=-e__ta_entry");
+    println!("cargo:rustc-link-arg=-pie");
+    println!("cargo:rustc-link-arg=-Os");
+    println!("cargo:rustc-link-arg=--sort-section=alignment");
+    println!("cargo:rustc-link-arg=--dynamic-list=dyn_list");
```

Comment:
One could probably move the boilerplate code into a crate that is only used as dev-dependencies to avoid the duplication.

Reply:
Sure, I will look for ways to move this into `optee-utee` crate.